clcache.py

```diff
@@ -230,16 +245,14 @@ def __init__(self, mutexName, timeoutMs):
             wintypes.INT(0),
             mutexName)
         self._timeoutMs = timeoutMs
-        self._acquired = False
+        self._mutexName = mutexName
```

Comment:
Can we move this up to avoid reusing paremeter symbol `mutexName`?

```python
    def __init__(self, mutexName, timeoutMs):
        self._mutexName = 'Local\\' + mutexName
        self._timeoutMs = timeoutMs

        self._mutex = windll.kernel32.CreateMutexW(
            wintypes.INT(0),
            wintypes.INT(0),
            self._mutexName)
        assert self._mutex
```